service/rest-service/proxy-rest-service/src/main/java/uk/gov/gchq/palisade/rest/ProxyRestService.java

```diff
@@ -51,8 +53,15 @@
     public static final String VERSION = "v1";
     private static final Logger LOGGER = LoggerFactory.getLogger(ProxyRestService.class);
     private static final String URL_CONF_KEY_SUFFIX = ".proxy.rest.url";
+    private static final String NUM_RETRIES_KEY = ".proxy.retry.max";
+    private static final String RETRY_WAIT_TIME = ".proxy.retry.pause";
```

Comment:
I would rename this to be clear that it is a key and not a value RETRY_WAIT_TIME_KEY